crates/forge/tests/cli/zk_cmd.rs

```diff
@@ -122,3 +122,57 @@ contract CallEmptyCode is Test {
         .stdout_lossy()
         .contains("call may fail or behave unexpectedly due to empty code");
 });
+
+// Sending ETH to an EOA doesn't trigger empty code warning
+forgetest_async!(test_zk_can_send_eth_to_eoa, |prj, cmd| {
```

Comment:
```suggestion
forgetest_async!(test_zk_can_send_eth_to_eoa_without_warnings, |prj, cmd| {
```
and delete comment above